Makefile

```
@@ -68,6 +68,7 @@ clean:
 
 generate:
 	GOARCH=$(NATIVE_ARCH) go generate ./...
+	(cd support && ./generate.sh)
```

Should the user see the gory details? If not then
```suggestion
	@(cd support && ./generate.sh)
```

---

As the added output when running `make agent` is just
```
(cd support && ./generate.sh)
```
I prefer to keep it that way. So when looking into the build process, we can make sure it runs and the verbosity of this single added line is acceptable.